client/checkout/classic/event-handlers.js

```diff
@@ -239,71 +244,15 @@ jQuery( function ( $ ) {
 		if ( hasPaymentMethodCountryRestrictions( upeElement ) ) {
 			togglePaymentMethodForCountry( upeElement );
 
-			// this event only applies to the checkout form, but not "place order" or "add payment method" pages.
-			$( '#billing_country' ).on( 'change', function () {
-				togglePaymentMethodForCountry( upeElement );
-			} );
-		}
-	}
-
-	function isBillingInformationMissing() {
```

Comment:
moving to `upe.js` as it's clearly a helper function

Reply:
Ah yeah, I can remember this topic now that you were bringing it up some time ago. Thanks, that makes sense, I'll leave it here for now.